q2cli/builtin/tools.py

```diff
@@ -16,7 +16,6 @@
 
 _COMBO_METAVAR = 'ARTIFACT/VISUALIZATION'
 
-
```

Comment:
I like to try and keep my diffs as minimal as possible - I think it helps future readers of the code (especially when they are using a tool like git). A "no-op" change like this can create noise (in my opinion). Can you undo the deletion of this line?